code/index.html

```diff
@@ -5,8 +5,69 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Project 4: Pizzabot</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="icon" type="image/png" href="./img/pizza.png" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.0.2/jquery.min.js"></script>
   </head>
-  <body>
-    <h1>This is your Pizza Bot</h1>
+ 
+<body id="orderPlaced">
+
+    <!-- H1 NEON TITLE   -->
+    <div class="neon">
+      <h1 class="green">This is your Pizza Bot</h1>
+    </div>
+    
+    <!-- CHATBOT FORM -->
+    <section class="chatbot-form">
```

Comment:
This whole section has a lot of wrong indents, moreover I saw couple places where you open `div`, but there is no closing tag - that needs to be fixed